inc/ocf_logger.h

```diff
@@ -31,7 +31,7 @@ typedef enum {
 struct ocf_logger_ops {
 	int (*open)(ocf_logger_t logger);
 	void (*close)(ocf_logger_t logger);
-	int (*printf)(ocf_logger_t logger, ocf_logger_lvl_t lvl,
+	int (*print)(ocf_logger_t logger, ocf_logger_lvl_t lvl,
 			const char *fmt, va_list args);
 	int (*printf_rl)(ocf_logger_t logger, const char *func_name);
```

Comment:
I would change also printf_rl() to print_rl() to have consistent naming(). Right now we have print() and print**f**_rl()